backend/config/index.ts

```diff
@@ -2,13 +2,30 @@ import dotenv from 'dotenv'
 
 dotenv.config()
 
-// TODO: have all the environment variables in one place and import this variable everywhere
-// like we did with the brilliantly named "my-project"
 export const config = {
-  mongoURI: process.env.MONGO_URI
+  mongoURI: process.env.MONGO_URI || 'mongodb://127.0.0.1:27017',
+  shouldServeReactApp: process.env.SERVE_REACT_APP === 'true',
+  port: process.env.PORT,
+  environment: process.env.NODE_ENV,
+  jwtSecret: process.env.JWT_SECRET
 }
 
-// TODO: throw an error if the environment variable is not set
 if (!config.mongoURI) {
   throw new Error('MONGO_URI environment variable is not set')
 }
+
+if (!config.shouldServeReactApp) {
```

Comment:
Either you didn't test the application locally or you set this to be "true" in your `.env` file, both options are wrong lol